pc-service/src/main/java/com/pokemaster/controller/TeamController.java

```diff
@@ -0,0 +1,29 @@
+package com.pokemaster.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.pokemaster.model.Team;
+import com.pokemaster.repository.TeamRepository;
+
+@RestController("teamController")
+@RequestMapping("/team")
```

Comment:
```suggestion
@RequestMapping("pc/team")
```
For the sake of microservices, pc should be appended as it is coming from the pc service. This might change later, but it's easier for now.